avaframe/runScripts/runComputeDFAPath.py

```diff
@@ -17,65 +19,112 @@
+        # load DFA results (use runCom1DFA to generate these results for example)
+        # here is an example with com1DFA but another DFA computational module can be used
+        # as long as it produces some particles or FV, FD and FM results
+        # create dataFrame of results (read DFA data)
+        simDF, _ = cfgUtils.readAllConfigurationInfo(avalancheDir)
 
-
-DFAPathCfgFile = pathlib.Path('ana5Utils', 'DFAPathGenerationCfg.ini')
-DFAPathCfg = cfgUtils.getModuleConfig(DFAPath)
-for simName, simDFrow in simDF.iterrows():
-    log.info('Computing avalanche path from simulation: %s', simName)
-    pathFromPart = DFAPathCfg['PATH'].getboolean('pathFromPart')
-    avaProfileMass = DFAPath.generateAveragePath(avalancheDir, pathFromPart, simName, dem)
-    # here the avaProfileMass given in input is overwriten and returns only a x, y, z extended profile
-    avaProfileMass = DFAPath.extendDFAPath(avalancheDir, DFAPathCfg['PATH'], dem, simName, avaProfileMass)
-    # put path back in original location
-    avaProfileMass['x'] = avaProfileMass['x'] + dem['originalHeader']['xllcenter']
-    avaProfileMass['y'] = avaProfileMass['y'] + dem['originalHeader']['yllcenter']
-    # save profile in Inputs
-    pathAB = pathlib.Path(avalancheDir, 'Outputs', 'DFAPath', 'massAvgPath_%s_AB_aimec' % simName)
-    name = 'massAvaPath'
-    shpConv.writeLine2SHPfile(avaProfileMass, name, pathAB)
-    log.info('Saved path to: %s', pathAB)
+    DFAPathCfg = cfgUtils.getModuleConfig(DFAPath)
+    for simName, simDFrow in simDF.iterrows():
+        log.info('Computing avalanche path from simulation: %s', simName)
+        pathFromPart = DFAPathCfg['PATH'].getboolean('pathFromPart')
+        resampleDistance = DFAPathCfg['PATH'].getfloat('nCellsResample') * dem['header']['cellsize']
+        # get the mass average path
+        avaProfileMass, particlesIni = DFAPath.generateAveragePath(avalancheDir, pathFromPart, simName, dem)
+        avaProfileMass, _ = gT.prepareLine(dem, avaProfileMass, distance=resampleDistance, Point=None)
+        avaProfileMass['indStartMassAverage'] = 1
+        avaProfileMass['indEndMassAverage'] = np.size(avaProfileMass['x'])
+        # make the parabolic fit
+        parabolicFit = DFAPath.getParabolicFit(DFAPathCfg['PATH'], avaProfileMass, dem)
+        # here the avaProfileMass given in input is overwriten and returns only a x, y, z extended profile
+        avaProfileMass = DFAPath.extendDFAPath(DFAPathCfg['PATH'], avaProfileMass, dem, particlesIni)
+        # resample path and keep track of start and end of mass averaged part
+        avaProfileMass = DFAPath.resamplePath(DFAPathCfg['PATH'], dem, avaProfileMass)
+        # get split point
+        splitPoint = DFAPath.getSplitPoint(DFAPathCfg['PATH'], avaProfileMass, parabolicFit)
+        # make analysis and generate plots
+        errorEnergyTest = outCom3Plots.generateCom1DFAPathPlot(avalancheDir, DFAPathCfg['PATH'], avaProfileMass, dem,
```

Comment:
```suggestion
        _ = outCom3Plots.generateCom1DFAPathPlot(avalancheDir, DFAPathCfg['PATH'], avaProfileMass, dem,
```

Reply:
as this parameter is not used and I was confused why the energyTest pops up here..